srcalc/src/main/webapp/js/enterVariables.js

```diff
@@ -63,15 +68,20 @@ var ENTERVARIABLES = function() {
             });
             
             // Get a DataTables API instance
-            var apiTable = proceduresTable.dataTable().api();
+            var apiTable = procedureTable.dataTable().api();
+            // Find the input within the container div.
+            var searchInput = $(apiTable.table().container()).
+                find('div.dataTables_filter input');
             // Unbind the default global search and keyup
-            $("div.dataTables_filter input").unbind();
+            searchInput.unbind();
             // Add a "starts with" regex to the global search
             // Enable regex search and disable smart searching
-            $("div.dataTables_filter input").on('keyup', function () {
+            searchInput.on('keyup', function () {
                     apiTable.column(0).search('^' + this.value, true, false).draw();
             });
 	    
+            // The table is done rendering.
+            procedureSelectDialog.removeClass('uninitialized');
```

Comment:
I know this means that all of the necessary datatables-related things have been done, but is there any state where we would get to this line without initializing something?